unit1-intro-to-programming-in-javascript/README.md

```diff
@@ -16,6 +16,7 @@
   - Attributes: class, name
   - Common CSS Properties: margin, padding, position, color, font-size, bg-color, etc.
   - CSS selectors
+  - CSS Layouts crash course: display flex, flex-direction, align-items, justify-content, etc
```

Comment:
If you insist. I would prefer to cut even intermediate CSS and instead focus on actual programming, especially if we are strapped for time. I just think CSS is quirky and everyone does it a bit differently (using frameworks, `less`, or something else)

However if we just want to expose them to it not dedicate too much time to it, i have no explicit objection.